modules/eco-node-maintenance-operator-installation-web-console.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-node-maintenance-operator-using-web-console_{context}"]
+= Installing the Node Maintenance Operator by using the web console
+
+You can use the {product-title} web console to install the Node Maintenance Operator.
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Operators* -> *OperatorHub*.
+. Search for the Node Maintenance Operator, then click *Install*.
+. Keep the default selection of *Installation mode* and *namespace* to ensure that the Operator will be installed to the `openshift-operators` namespace.
+. Click *Install*.
+
+.Verification
+
+To confirm that the installation is successful:
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+. Check that the Operator is installed in the `openshift-operators` namespace and that its status is `Succeeded`.
+
+If the Operator is not installed successfully:
+
+. Navigate to the *Operators* -> *Installed Operators* page and inspect the `Status` column for any errors or failures.
+. Navigate to the *Workloads* -> *Pods* page and check the logs in any pods in the `openshift-operators` project that are reporting issues.
```

Comment:
`openshift-operators` is a namespace, and we have been referring him as namespace in the other occurrences.

Reply:
Or maybe it is ok, since you might have referred to the project on the top left (where you select the namespace).